Week01/Problem03/syucel_03.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Jul 15 19:28:32 2018
+
+@author: sezen
+"""
+
+#lpf = largest prime factor
+def lpf(x):
+    
+    factors = []
+    factor = 2
+    while factor <= x:
+        if x % factor == 0:
+            x=x/factor
+            factors.append(factor)
+        else:
+            factor = factor + 1
```

Comment:
For truly giant numbers (on the order of 600851475143^2), looping in a "bottom-up" approach is probably not the way to go for your loop...can you think of a way to reverse it, i.e., start from the largest number the lpf **could** be, then work backwards?